code/online_challenges/src/project_euler/problem_025/problem_025.cpp

```diff
@@ -0,0 +1,38 @@
+#include <iostream>
+#include <vector>
+using namespace std;
+
+int main() 
+{
+    vector < int > temp, prevFibonacci, currFibonacci;
+
+    int count = 2;
+
+    prevFibonacci.push_back(1);
+    currFibonacci.push_back(1);
+        
+    while (currFibonacci.size() < 1000) 
+    {
+        int carry = 0;
+        count++;
+        for (int i = 0; i < prevFibonacci.size(); i++) 
```

Comment:
I would use [`std::size_t`](http://en.cppreference.com/w/cpp/types/size_t) for a loop index in all of your for loops. (TBH it doesn't really matter here)